### ISSUE TITLE
Compilations fails when using serve -c

### ISSUE THREAD

**Issue by Deraen** [karthikv/nodefront#29]
When serve() calls compile()  (at line 38 file serve.js) it doesn't specify output-property for env-parameter.
- compile() calls generateCompileFn and uses env.output (undefined) for directory parameter
- at line 244 generateCompileFn uses the directory parameter but as it is undefined:
  
  ```
   Compilation error for css/main.less: ENOENT, open 'undefined/css/main.css'
  ```

**Comment by karthikv** [comment]
Ah, yes. Good catch; I'll get this fixed soon.

**Comment by karthikv** [comment]
Apologies for the delay; this has been addressed in the latest 1.0.4 release (see commit above). Let me know if you continue to experience issues with the updated release.
